### PR TITLE
Fix windows commit installs

### PR DESCRIPTION
**Pull request by dereknola**
Signed-off-by: Derek Nola <derek.nola@suse.com>

#### Proposed Changes ####
Add missing `-Not` to two if checks (Parameter was always passed and we WANT to check if it is empty)
Checks if `/var/lib/rancher/rke2/agent/images` directory exists before trying to make a new one
Removes duplicate directory creation line

#### Types of Changes ####
Bugfix

#### Verification ####
Setup RKE2 linux server with a commit 
`curl -sfL https://get.rke2.io | INSTALL_RKE2_COMMIT=<COMMIT> sh -`

Install RKE2 on windows with commit 
My yaml config
```
kube-proxy-arg: "feature-gates=IPv6DualStack=false"
node-external-ip: #{win_agent_ip}
node-ip: #{win_agent_ip}

server: https://#{server_ip}:9345
token: vagrant-rke2
```
`./install.ps1 -Commit <COMMIT>` now installs correctly, and RKE2 agent will connect to linux server
Additionally, running the `install.ps1` again will no longer break complaining about an existing /var/lib/rancher/rke2/agent/images directory

#### Linked Issues ####
https://github.com/rancher/rke2/issues/2543
https://github.com/rancher/rke2/issues/2125

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -635,22 +635,24 @@ function Install-AirgapTarball() {
...
+    if (-Not $ExpectedImageAirGapChecksum) {
         return
     }
 
     $archInfo = Get-ArchitectureInfo
     $suffix = $archInfo.Suffix    
     $arch = $archInfo.Arch
 
+    if (Test-Path $InstallAgentImageDir) {
+        Remove-Item -Path $InstallAgentImageDir -Force -Recurse
```
Comment (brandond) [inline install.ps1:647]: Why are we nuking the agent image dir? What if the user had placed some of their own image tarballs there prior to install?

**Review by phillipsj — APPROVED** [review]

File: install.ps1 (hunk 1 above)
Comment (phillipsj) [reply install.ps1:647]: Yeah, this is new behavior as compared to the Linux install script.

**Review by brandond — APPROVED** [review]

**Review by phillipsj — APPROVED** [review]
LGTM!

File: install.ps1 [hunk 2]
```diff
@@ -600,7 +600,7 @@ function Test-AirgapTarballChecksum() {
         return
     }
 
-    if ($ExpectedImageAirGapChecksum) {
+    if (-Not $ExpectedImageAirGapChecksum) {
```
Comment (rosskirkpat) [inline install.ps1:603]: The install script in master is working based on my testing https://github.com/rancher/rke2/issues/2543#issuecomment-1062217804, have you verified that you are running powershell as an administrator?

**Review by rosskirkpat — CHANGES_REQUESTED** [review]

File: install.ps1 (hunk 2 above)
Comment (rosskirkpat) [reply install.ps1:603]: I had an incorrect understanding of the issue, removing my block and approving as I have validated this fix myself as well.

**Review by rosskirkpat — APPROVED** [review]

**Comment by rosskirkpat** [comment]
Thank you for your patience while I figured this one out @dereknola! It's greatly appreciated

**Review by rosskirkpat — CHANGES_REQUESTED** [review]
blocking this as there are lower level issues that cause these changes to not work due to using zstd compressed tarballs for the dev-runtime-image outputs.

I am taking the design issue offline with @dereknola

**Review by rosskirkpat — APPROVED** [review]
LGTM

**Pull request merged**